rest-api/tools/import_organizations.py

```diff
@@ -348,6 +348,11 @@ def _populate_lat_lng_and_time_zone(self, site, existing_site):
         site.longitude = longitude
         if latitude and longitude:
           site.timeZoneId = self._get_time_zone(latitude, longitude)
+      else:
```

Comment:
The flag documentation should make mention of this or else flag name changed (e.g. --stub_geocoding). Not obvious from this that `--gecode_flag` would exclusively be passed in non-testing environments.

Reply:
Switching to --stub_geocoding